src/utils.tsx

```diff
@@ -270,3 +271,22 @@ export const codeShouldBeTrimmed = (
 ) => {
   return codeLength >= slowLoadingCharCount || isMinified;
 };
+
+export const sendPerfTiming = ({
+  _tracking = tracking,
+  actualDuration,
+  id,
+  phase,
+}: {
+  _tracking?: typeof tracking;
+  actualDuration: number;
+  id: string;
+  phase: string;
+}) => {
+  _tracking.timing({
+    category: 'renderPerf',
+    label: phase,
+    value: actualDuration,
+    variable: id,
+  });
```

Comment:
How is it looking in GA? Can we graph these metrics? Or can we use aggregate functions at least (to compute percentiles, etc.)?

Reply:
We have data in GA now for all the fetch timings and I've taken a look at that and it does look like we can manipulate it in useful ways. You can take a look too, but you might need to ask for access to the code manager sites in GA.